src/webui/components/Help/index.js

```diff
@@ -1,42 +1,50 @@
+/**
+ * @prettier
+ * @flow
+ */
+
 import React from 'react';
-import SyntaxHighlighter, {registerLanguage} from 'react-syntax-highlighter/dist/light';
-import sunburst from 'react-syntax-highlighter/src/styles/sunburst';
-import js from 'react-syntax-highlighter/dist/languages/javascript';
+import { CardStyled as Card } from './styles';
+import CardActions from '@material-ui/core/CardActions/index';
+import CardContent from '@material-ui/core/CardContent/index';
+import Button from '@material-ui/core/Button/index';
+import Typography from '@material-ui/core/Typography/index';
+
+import CopyToClipBoard from '../CopyToClipBoard/index';
 
-import classes from './help.scss';
-import {getRegistryURL} from '../../utils/url';
+import { getRegistryURL } from '../../utils/url';
 
-registerLanguage('javascript', js);
+import type { Node } from 'react';
+import { IProps } from './types';
 
-const Help = () => {
-  const registryURL = getRegistryURL();
+const Help = ({ scope = '' }: IProps): Node => {
+  const registryUrl = getRegistryURL();
 
-    return (
-      <div className={classes.help}>
-        <li className={classes.noPkg}>
-          <h1 className={classes.noPkgTitle}>
-            No Package Published Yet
-          </h1>
-          <div className={classes.noPkgIntro}>
-            <div>
-              To publish your first package just:
-            </div>
-            <br/>
-            <strong>
-              1. Login
-            </strong>
-            <SyntaxHighlighter language='javascript' style={sunburst} id="adduser">
-              {`npm adduser --registry  ${registryURL}`}
-            </SyntaxHighlighter>
-            <strong>2. Publish</strong>
-            <SyntaxHighlighter language='javascript' style={sunburst} id="publish">
-              {`npm publish --registry ${registryURL}`}
-            </SyntaxHighlighter>
-            <strong>3. Refresh this page!</strong>
-          </div>
-        </li>
-      </div>
-    );
+  return (
+    <Card>
+      <CardContent>
+        <Typography component="h2" variant="headline" gutterBottom>
+          No Package Published Yet.
+        </Typography>
+        <Typography color="textSecondary" gutterBottom>
+          To publish your first package just:
+        </Typography>
+        <br />
+        <Typography variant="body2" gutterBottom>
```

Comment:
To follow the same style, I'd follow the first `1.` point structure. 

I think you can extract a method and avoid repetition here. 

```
      <CopyToClipBoard text={xxxxx} />
        <Typography variant="body2">xxxx</Typography>
```

Reply:
Done !!